crates/ruff_cli/src/cache.rs

```diff
@@ -58,6 +66,7 @@ impl Cache {
     ///
     /// Finally `settings` is used to ensure we don't open a cache for different
     /// settings.
+    #[allow(clippy::cast_possible_truncation)]
```

Comment:
Nit: Can we add a `// SAFETY` comment explaining why the truncation is okay in this case?